### PR TITLE
Add more descriptive description and download button when statistics not available

### PR DESCRIPTION
**Pull request by peppy**
Before:

image

After:

### CODE REVIEW

File: osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs [hunk 1]
```diff
@@ -75,7 +75,21 @@ private void populateStatistics(ValueChangedEvent<ScoreInfo> score)
                 return;
 
             if (newScore.HitEvents == null || newScore.HitEvents.Count == 0)
-                content.Add(new MessagePlaceholder("Score has no statistics :("));
```
Comment (smoogipoo) [inline osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs:78]: Please wrap this in a `{ }`

```
if (...)
    x.Add(
    {
        ...
    }
else
{
...
}
```
Is almost bound to lead to errors.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

File: osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs:78]: you mean just adding a pair of `{}` for the outer if, correct?

Comment (smoogipoo) [reply osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs:78]: yes

Comment (peppy) [reply osu.Game/Screens/Ranking/Statistics/StatisticsPanel.cs:78]: it's actually enforced by our style guidelines, overlooked.

**Review by smoogipoo — DISMISSED** [review]

**Pull request merged**